### DISCUSSION TITLE
zh/docs/get-started

### DISCUSSION THREAD

**Discussion by giscus** [casbin/casbin#1147] [Docs comments]
# zh/docs/get-started

开始使用 Casbin

https://docs.casbin.cn/zh/docs/get-started

**Comment by tianya441523** [comment]
Original error point：
results, err := e.BatchEnforce([[] []interface{}{"alice", "data1", "read"}, {"bob", datata2", "write"}, {"jack", "data3", "read"}})

Should be changed to:
results, err := e.BatchEnforce([][]interface{}{{"alice", "data1", "read"}, {"bob", "data2", "write"}, {"jack", "data3", "read"}})

**Reply by Selflocking** [reply]
This is due to a translation error, it has been fixed

**Comment by oldmee** [comment]
a, err := xormadapter.NewAdapter("mysql", "mysql_username:mysql_password@tcp(127.0.0.1:3306)/casbin")

should be:
a, err := xormadapter.NewAdapter("mysql", "mysql_username:mysql_password@tcp(127.0.0.1:3306)/casbin", true)

**Reply by oldmee** [reply]
or:
a, err := xormadapter.NewAdapter("mysql", "mysql_username:mysql_password@tcp(127.0.0.1:3306)/")

**Reply by hsluoyz** [reply]
@oldmee fixed in: https://github.com/casbin/casbin-website-v2/pull/191

**Comment by nishantbadhautiya** [comment]
postgresql casbin adaptor not working with postgres v7

**Reply by nishantbadhautiya** [reply]
I mean with prisma
